bokehjs/src/vendor/bootstrap-3.1.1/less/panels.less

```diff
@@ -21,8 +21,10 @@
 // Optional heading
 .bk-bs-panel-heading {
   padding: 10px 15px;
-  border-bottom: 1px solid transparent;
-  .bk-bs-border-top-radius((@panel-border-radius - 1));
+  // border-bottom: 1px solid transparent;
+  // .bk-bs-border-top-radius((@panel-border-radius - 1));
+  border: #cc0000;
+  background-color: pink;
```

Comment:
`bokehjs/src/vendor` shouldn't be modified unless there is no way around it. Changes like this should be placed in `bokejs/src/less/overrides.less`.